python/rikai/spark/functions/vision.py

```diff
@@ -280,3 +281,22 @@ def spectrogram_image(
         format=image_format,
         **image_kwargs,
     )
+
+
+@udf(returnType=ArrayType(IntegerType()))
+def img_cluster(img_list, threshold=0.5, img_shape=(300, 300)):
```

Comment:
@smellslikeml does keyword arguments work? like you can't do `img_cluster("image_col", img_shape=(400, 300))` can you?

Reply:
Just need the dimensions of reference and test images to be the same for SSIM.